watertap/examples/flowsheets/lsrro/tests/test_lsrro.py

```diff
@@ -256,16 +260,16 @@ class TestLSRRO_1Stage(_TestLSRRO):
 
     number_of_stages = 1
 
-    number_of_variables = 291
-    number_of_constraints = 190
+    number_of_variables = 304
+    number_of_constraints = 193
 
     display_system = \
 """----system metrics----
 Feed: 1.00 kg/s, 70000 ppm
 Product: 0.180 kg/s, 1592 ppm
 Volumetric water recovery: 18.0%
 Energy Consumption: 3.8 kWh/m3
-Levelized cost of water: 2.48 $/m3
+Levelized cost of water: 1.73 $/m3
```

Comment:
This looks like a pretty significant change in LCOW. Was this because you factored in the recovered energy from the ERD that may have been omitted previously in this result (i.e., subtracted electricity cost due to energy recovery)?

Reply:
The specific energy consumption, without the ERD, is something like 13-14 kWh/m^3 for this process. I didn't carefully cross-check this; the reduction in LCOW seem about right to me.

The correct way to do this would be to fix the original bug first so we could have a new baseline. Discovering it in process here makes doing proper cross-validation more difficult.